lua/telescope/builtin/init.lua

```diff
@@ -311,13 +311,23 @@ builtin.lsp_dynamic_workspace_symbols = require('telescope.builtin.lsp').dynamic
 ---   - `<C-l>`: show autocompletion menu to prefilter your query with the diagnostic you want to see (i.e. `:warning:`)
 ---@param opts table: options to pass to the picker
 ---@field hide_filename boolean: if true, hides the name of the file in the current picker (default is false)
+---@field severity string or number: filter diagnostics by severity name or id (i.e. 'error' or 1)
```

Comment:
`string or number` is not a thing in the grammar. Its `string|number|function|table`. Thats probably breaking `docgen` output. (No idea if `docgen` is handling multiple types tho :thinking:)

Reply:
Yeah, does not seem to work. I've not set the type to `string` and added a note for severity fields that they take one of strings or numbers as per `vim.lsp.protocol.DiagnosticSeverity`.

I hope that's fine.

Reply:
![image](https://user-images.githubusercontent.com/15233006/120180124-af83a080-c20b-11eb-9e9d-c9b42249f01a.png)

Treesitter grammar understands it so only the `docgen` module is missing support. I can add this, that week

Reply:
Ok cool, then let's hold off on this up until then.

Reply:
https://github.com/tjdevries/tree-sitter-lua/pull/26